docs/win32.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Windows Support
+---
+
+## Introduction
+
+The Windows MSI package is built automatically in continuous integration.
+Only some plugins are included, any plugins that use Linux-specific features like `udev`, `efivarfs` or `devfs` are disabled.
+However, some USB devices updatable in Linux are also updatable in Microsoft Windows and macOS too.
+
+## Installation
+
+First, install the `.msi` package from the [release announcements](https://github.com/fwupd/fwupd/releases/), or any of the CI artifacts such as `dist/setup/fwupd-1.9.10-setup-x86_64.msi`.
+
+Then click **More info** and **Run anyway** buttons:
```

Comment:
Probably should explain here that it's not signed because it's built by CI.  Although; should we be putting a secret in CI and signing it?  I don't know if that's feasible or not to sign a windows binary from Linux.

Reply:
> should we be putting a secret in CI and signing it

If I understand correctly, that's a signing cert, which is $$$ per year -- and we could never use that in CI, it'd have to be signed locally on a private machine (ideally with an HSM attached).